packages/moonstone/VideoPlayer/VideoPlayer.js

```diff
@@ -1962,6 +2005,32 @@ const VideoPlayerBase = class extends React.Component {
 		const moreDisabled = !(this.state.more);
 		const controlsAriaProps = this.getControlsAriaProps();
 
+		const initialPlayingProps = {
```

Comment:
Could you add some documentation to describe the difference between these two sets of props?  Also, should we apply keys to either of these in case the app doesn't set a preload video immediately?